recipes/aadict/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "aadict" %}
+{% set version = "0.2.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aadict-{{ version }}.tar.gz
+  sha256: a77328ac55dbb5735da99441870251befe135f687ab707a7a178561363b27704
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - six >=1.6.0
+
+test:
+  imports:
+    - aadict
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: http://github.com/metagriffin/aadict
+  summary: An auto-attribute dict (and a couple of other useful dict functions)
+  license: GPL-2.0-or-later
```

Comment:
Looks like this is actually 3.0 or later

Reply:
thanks, fixed!